sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -1224,7 +1224,10 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
   test("SPARK-33084: Add jar support Ivy URI in SQL") {
     val testData = TestHive.getHiveFile("data/files/sample.json").toURI
     withTable("t") {
-      sql(s"ADD JAR ivy://org.apache.hive.hcatalog:hive-hcatalog-core:$hiveVersion")
+      // exclude org.pentaho:pentaho-aggdesigner-algorithm as this transitive dependency does
+      // not exist on mavencentral and hence cannot be found in the test environment
+      sql(s"ADD JAR ivy://org.apache.hive.hcatalog:hive-hcatalog-core:$hiveVersion" +
+        "?exclude=org.pentaho:pentaho-aggdesigner-algorithm")
```

Comment:
This is odd. Any idea why `hive-hcatalog-core` has a dependency on a package that doesn't exist?

Reply:
This is a very infamous dependency brought in by calcite. It does not exist on mavencentral but it does exist on conjars.org. Here is a Stack overflow asking about the same issue. https://stackoverflow.com/questions/32587769/dependency-resolution-issue-in-gradle

Reply:
Yikes.